datalad/core/distributed/tests/test_clone.py

```diff
@@ -51,6 +51,7 @@
     ok_file_has_content,
     assert_not_in,
     assert_raises,
+    assert_re_in,
```

Comment:
This import is no longer needed with the `assert_logged` switch.  (Sorry, for unintentionally starting all this trouble way back in [3518][0]!  I really don't have a strong preference between using `assert_logged` or `assert_re_in`.)

Given the current testing run is well underway, I'd say it'd be fine to drop this import once the tests complete and then do a merge without the full test cycle.

[0]: https://github.com/datalad/datalad/pull/3518#discussion_r301685701

Reply:
Thx!